package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "angel_arena_allstars",
-  "version": "0.0.0",
+  "name": "open_angel_arena",
+  "version": "0.0.1",
```

Comment:
we're not at 0.1 yet, and we'd use npm cli to do it anyway (for auto-tagging and all that fanciness)